components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -121,6 +121,16 @@ Configure the OpenSearch `path.repo` attribute.
     chef-automate backup create
     ```
 
+### Pre-Restore Validation
+
+Before executing the actual restore command, it is recommended to use the --verify-restore-config flag. This flag performs a pre-check to ensure that all backup and restore-related configurations are correctly set.
+Running this verification helps identify potential issues in advance, allowing users to resolve them before initiating the restore process. If these issues are only discovered during the restore operation, they can be harder to diagnose, as errors appear only in logs after the restore attempt fails.
+Using this flag proactively minimizes the risk of restore failures and ensures a smoother, more efficient, and reliable restore process.
```

Comment:
Replace it with: This flag proactively minimizes the risk of restore failures and ensures a smoother, more efficient, and reliable restore process.